roles/node/templates/testnet/config.toml.j2

```diff
@@ -19,6 +19,8 @@ eth_block_gas_limit = 84000000
 gas_price = "4_761_904_800_000"
 scilla_address = "http://localhost:{{ zilliqa.scilla.ports.scilla_port }}"
 scilla_stdlib_dir = "{{ node_scilla_lib_dir }}"
+scilla_server_socket_directory= "{{ node_socket_dir }}"
```

Comment:
At https://github.com/encapsulate-xyz/zilliqa-ansible/blob/main/roles/scilla/templates/systemd.service.j2#L9, please see if this works to add the timestamp

```
ExecStart=/bin/sh -c '/opt/zilliqa-scilla/bin/zilliqa-scilla --port 62831 --verbose \
                      2>&1 | /usr/bin/ts "[%Y-%m-%d %H:%M:%S]"'
```

Feel free to try something else if that doesnt work